test/issue-466-test.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
```

Comment:
Stuff the tests inside of this into a suitably named new describe block in `fake-timers-test.js`. We usually reserve the `issues*` files for regression test cases.